coverage_conditional_plugin/__init__.py

```diff
@@ -4,7 +4,10 @@
 from importlib import import_module
 from typing import ClassVar, Dict, Iterable, Optional, Tuple, Union
 
-import pkg_resources
+try:  # pragma: no cover
+    from importlib.metadata import version as metadata_version
+except ImportError:  # pragma: no cover
+    from importlib_metadata import version as metadata_version  # type: ignore
```

Comment:
Unclear why mypy fails on this.
I'm also unable to run mypy locally because it attempts to check my entire system's python installation with the provided mypy configuration :S

Reply:
I get:

```
poetry run mypy coverage_conditional_plugin
coverage_conditional_plugin/__init__.py:10: error: Name "metadata_version" already defined (possibly by an import)
Found 1 error in 1 file (checked 1 source file)
```

Reply:
See https://github.com/wemake-services/coverage-conditional-plugin/pull/177#discussion_r979825020